common/stub-setup.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Injects the stubs into the global scope.
+ * Bundled HomeApp cannot be loaded until these globals are set.
+ */
+import { DeviceManagerStub } from './device-manager';
+import { AppStub } from './smart-home-app';
+
+(global as any).smarthome = {
+  App: AppStub,
+  DeviceManager: DeviceManagerStub,
+};
+
+//TODO(cjdaly) find a cleaner way to do this.
+export function loadHomeApp() {
+  require('../home-app/bundle');
```

Comment:
Does this implicitly return the bundle?

Reply:
Can you make it a command-line argument to the filepath of the bundled JS rather than having to copy it?

Reply:
> Does this implicitly return the bundle?

No, but it implicitly runs bundle.js.  I would like to find a better way to run the bundle so it's not misleading though.



> Can you make it a command-line argument to the filepath of the bundled JS rather than having to copy it?

Yeah this is a good idea I'll add it.

Reply:
Opened an issue for this #3, but may not need a CLI when packaging this project into a node module (see #21)